behave/run_test_local.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash
 export DOCKER_BUILDKIT=0
 
+while [ -n "$1" ]; do # while loop starts
+	case "$1" in
+	--database-snapshot-enabled)
+
+	  echo "Running with database snapshot enabled"
+	  export DATABASE_SNAPSHOT_ENABLED=True
+	;; # Message for -a option
+	esac
+	shift
+done
+
 docker-compose down
-docker-compose -f docker-compose.yml -f docker-compose.local.yml run start_applications
+docker-compose -f docker-compose.yml run start_applications
```

Comment:
does this need to come out now you have edited the postgres image in that file?

Reply:
I don't think we need the .local.yml any longer as the we are using the images from ecr